exchange/exchange_test.go

```diff
@@ -4073,6 +4074,132 @@ func TestAuctionDebugEnabled(t *testing.T) {
 
 }
 
+func TestExperimentConfigs(t *testing.T) {
+	//This test just pass experiment ads cert configs to HoldAuction
+
+	noBidServer := func(w http.ResponseWriter, r *http.Request) { w.WriteHeader(204) }
+	server := httptest.NewServer(http.HandlerFunc(noBidServer))
+	defer server.Close()
+
+	cfg := &config.Configuration{
+		Adapters: make(map[string]config.Adapter, 1),
+	}
+	cfg.Adapters["appnexus"] = config.Adapter{
+		Endpoint: "test.com", //Note the '&' character in there
```

Comment:
Is this a left over copy/paste? I don't see an & character.

Reply:
Copy paste, yes, sorry, fixed.